.github/workflows/pr-e2e.yml

```diff
@@ -138,7 +138,7 @@ jobs:
           TEST_CLUSTER_NAME: keda-pr-run
         run: |
           MESSAGE="${{ github.event.comment.body }}"
-          REGEX='/run-e2e (.+)'
+          REGEX='/run-e2e (.+\*)'
```

Comment:
We could want to give the whole test name, so we should allow also the file extension
```suggestion
          REGEX='/run-e2e (.+[\*|.go])'
```

Reply:
good idea!